faraday/aprs.py

```diff
@@ -103,6 +104,11 @@ def aprs_worker(config, sock):
     # Local variable initialization
     telemSequence = 0
 
+    #dev
+    sock = ''
```

Comment:
This line is unnecessary.

Reply:
Removed, `conn=False` now used in the initialization just above to force it to initialize on first loop. Thanks for catching this!